Combinatorial_map/include/CGAL/Combinatorial_map.h

```diff
@@ -759,84 +752,84 @@ namespace CGAL {
      * @return the dimension d such that dh is not d-free but k-free for
      *         all k>d. -1 if the dart is free for all d in {0..n}
      */
-    int highest_nonfree_dimension(Dart_const_handle dh) const
+    int highest_nonfree_dimension(Dart_const_descriptor dh) const
     {
       for (int i=(int)dimension; i>=0; --i)
       { if ( !is_free(dh, i) ) return i; }
       return -1;
     }
 
     /** Return a dart belonging to the same edge and to the second vertex
-     * of the current edge (nullptr if such a dart does not exist).
+     * of the current edge (null_descriptor if such a dart does not exist).
```

Comment:
```suggestion
     * of the current edge (`null_dart_descriptor` if such a dart does not exist).
```

Reply:
same change should be done everywhere, right?

Reply:
No (cf answer above)